common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -377,6 +377,16 @@ trait FeatureSwitches {
     exposeClientSide = false
   )
 
+  val LiveBlogAmpSwitch = Switch(
+    SwitchGroup.Feature,
+    "live-blog-amp-switch",
+    "If this switch is on, link to amp pages will be in the metadata for live blogs",
+    owners = Seq(Owner.withGithub("SiAdcock")),
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = true
```

Comment:
is it really needed client side? also normally we don't use "switch" in the switch name, just in the name of the val itself